rak811.go

```diff
@@ -24,17 +25,17 @@ type Lora struct {
 	timeout time.Duration
 }
 
+var defaultConfig = &serial.Config{
```

Comment:
This is not required, in case we need a different timeout it shouldn't be embedded in the serial default config (if its a different timeout.) and the setup for the tests can be passed in.

Reply:
sorry I am not sure what you mean by this.


Can you show some code suggestion